Team.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <iostream>
+#include <string>
+#include <iomanip>
+
+class Team
+{
+public:
+	std::string m_name;
+	double m_fifa_points, m_fifa_before;
+	int m_fifa_rang, m_place, m_games, m_points, m_goal_scored, m_goal_conceded, m_goal_difference;
+	Team(std::string name, int fifa_rang, double fifa_points);
+	~Team() = default;
+	void CountingStats(int scored, int conceded, double ImportanceOfMatch, const double another_team_points);
```

Comment:
Unclear name, I suggest to use a verb.

Reply:
And const for non-reference arguments is usually useless, although VS sometimes recommends to put it for some reason. In every way, be consistent -- you have three different naming styles here for arguments (PascalCase, snake_case, and snake_case or camelCase) and some of them are const and some not, while it does not make any actual difference in a declaration.

Reply:
It can be called something like EvaluateStats, as far as I can tell.